tests/client/test_treeherder_client.py

```diff
@@ -511,5 +522,43 @@ def test_treeheder_auth(self, mock_time, mock_generate_nonce):
                                                "user=project&"
                                                "oauth_signature=DJe%2F%2FJtw7s2XUrciG%2Bl1tfJJen8%3D"))
 
+    @patch("treeherder.client.client.requests.get")
+    def test_get_job(self, mock_get):
+
+        mock_get.return_value = self._get_mock_response({
+            "meta": {"count": 3,
+                     "repository": "mozilla-inbound",
+                     "offset": 0},
+            "results": [{"jobDetail1": 1},
```

Comment:
it would probably be better to define a JOB_RESULTS array with the contents:

[{"jobDetail1": 1}, ... ], then use that for the assertions and comparisons.